prelude/src/skstore/Handle.sk

```diff
@@ -119,6 +119,17 @@ mutable class NonEmptyIterator<+T>(
       this.materializedIter,
     )
   }
+
+  mutable fun convert<V>(conv: T ~> V): mutable NonEmptyIterator<V> {
```

Comment:
I'm curious why you didn't call this `map`.

Reply:
Because a map function already supplied by the Iterator Ancestor, but the return value is an Iterator and not a NonEmptyIterator

Reply:
After investigation, the crucial point seems to be `~>` vs `->`, see https://github.com/SkipLabs/skdb/pull/380.